src/components/storage-upload-helper.js

```diff
@@ -0,0 +1,49 @@
+import { IMAGE_DOWN_SCALING_WIDTH } from '../domain/constants.js';
```

Comment:
I wonder whether we should put this file outside of the `components` folder and into some of the `utils` (as it is not a react component?

Reply:
I considered putting it in `src/utils`, but I had second thoughts because it's only for the client side. However I see we have `file-helper` already, only for the server side-so that's fine then. I'll move it 👍

Reply:
:+1: